res/css/_common.pcss

```diff
@@ -37,6 +37,20 @@ $container-border-width: 8px;
 $container-gap-width: 8px; /* only even numbers should be used because otherwise we get 0.5px margin values. */
 
 $timeline-image-border-radius: 8px;
+
+:root, html {
+    background-color: #15191e;
+}
+html {
+    transition: 1s;
+    transition-duration: 1s;
+}
+body {
```

Comment:
This is causing lint failures because all three of these selectors (`:root`, `html`, `body`) already exist below. Please merge into the existing selectors.